collect_app/src/test/java/org/odk/collect/android/utilities/WidgetAppearanceUtilsTest.java

```diff
@@ -77,4 +103,89 @@ public void getNumberOfColumnsTest() {
         when(formEntryPrompt.getAppearanceHint()).thenReturn("columns--10");
         assertEquals(1, WidgetAppearanceUtils.getNumberOfColumns(formEntryPrompt, null));
     }
+
+    @Test
+    public void isNoButtonsAppearance_returnsFalse_whenNoButtonsAppearanceIsNotFound() {
+        assertFalse(WidgetAppearanceUtils.isNoButtonsAppearance(formEntryPrompt));
+    }
+
+    @Test
+    public void isNoButtonsAppearance_returnsTrue_whenNoButtonsAppearanceIsFound() {
+        when(formEntryPrompt.getAppearanceHint()).thenReturn("NO-BUTTONS");
+        assertTrue(WidgetAppearanceUtils.isNoButtonsAppearance(formEntryPrompt));
+    }
+
+    @Test
+    public void isCompactAppearance_returnsFalse_whenCompactAppearanceIsNotFound() {
+        assertFalse(WidgetAppearanceUtils.isCompactAppearance(formEntryPrompt));
+    }
+
+    @Test
+    public void isCompactAppearance_returnsTrue_whenCompactAppearanceIsFound() {
+        when(formEntryPrompt.getAppearanceHint()).thenReturn("COMPACT");
+        assertTrue(WidgetAppearanceUtils.isCompactAppearance(formEntryPrompt));
+    }
+
+    @Test
+    public void useThousandSeparator_returnsFalse_whenThousandSepAppearanceIsNotFound() {
+        assertFalse(WidgetAppearanceUtils.useThousandSeparator(formEntryPrompt));
+    }
+
+    @Test
+    public void useThousandSeparator_returnsTrue_whenThousandSepAppearanceIsFound() {
+        when(formEntryPrompt.getAppearanceHint()).thenReturn("THOUSANDS-SEP");
+        assertTrue(WidgetAppearanceUtils.useThousandSeparator(formEntryPrompt));
+    }
+
+    @Test
+    public void isFrontCameraAppearance_returnsFalse_whenFrontCameraAppearanceIsNotFound() {
+        assertFalse(WidgetAppearanceUtils.isFrontCameraAppearance(formEntryPrompt));
+    }
+
+    @Test
+    public void isFrontCameraAppearance_returnsTrue_whenFrontAppearanceIsFound() {
```

Comment:
Please add more cases, you can have multiple in one test if they are related. Would be good to test lowercase appearances as well and different variation (mix of appearances).

Reply:
Sorry I missed that.